tests/test_iostream.cpp

```diff
@@ -11,6 +11,8 @@
 #include <pybind11/iostream.h>
 #include "pybind11_tests.h"
 #include <iostream>
+#include <thread>
+#include <atomic>
```

Comment:
(Really not important, but if you still need to make changes, you could still sort these stdlib includes. But I don't think CI will complain, so... eh.)